packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1804,6 +1807,20 @@ export interface Disposable {
     dispose(): void;
 }
 
+// @internal
+export interface _DomainPermissionsProps {
+    appName: string;
+    onTroubleshootingClick: () => void;
+    strings: DomainPermissionsStrings;
+}
+
+// @beta
+export interface DomainPermissionsStrings {
+    linkText: string;
+    mainText: string;
```

Comment:
use `primary` instead of `main` (this then matches secondary as a naming prefix)